Dockerfile

```diff
@@ -43,19 +79,14 @@ ARG PYTHON_VERSION=3.8.0
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt update &&\
-    apt install -y software-properties-common &&\
-    add-apt-repository ppa:openjdk-r/ppa
-
 RUN sed -i -e '/^deb-src/d' /etc/apt/sources.list &&\
     apt-get update &&\
     apt-get install -y \
       curl \
       dpkg \
       net-tools \
-      openjdk-11-jdk \
```

Comment:
Does the ubuntu package not contain the latest Java 11?

Reply:
> Does the ubuntu package not contain the latest Java 11?

Currently, the latest from the ppa is 11.0.5, which has security vulnerabilities.  See jira ticket OTL-562.